### PR TITLE
LoanFormComponent are refactored

### LINKED ISSUE #554: Refactor LoanFormComponent
## ✨ Feature request
refactor LoanFormComponent into composition api

### 📜 Acceptance Criteria
- [ ] all code are refactored.

### PR DESCRIPTION
**Pull request by TE4-Filip-Liljenberg**
## ⚙️ What issue did you implement or fix?
Fixes #554

## 📜 Summary
- Refactored LoanFormComponent
- fixed test

## 📝 Type of change

Please delete options that are not relevant.

- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/components/LoanFormComponent.vue [hunk 1]
```diff
@@ -54,49 +54,63 @@
...
-  // user to recreate a loan 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // current user that has been logged in and then rerenders the loan form for the 
+    // user to recreate a loan 
+    // user to recreate a loan 
+    // user to recreate a loan
```
Comment (te4-alexander-nylundgomes) [inline frontend/src/components/LoanFormComponent.vue:82]: Might wanna remove these comments, they seem unnecessary 
```suggestion
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // Eventlistener that does not reload the page when executed, sets the lent by id to the 
    // current user that has been logged in and then rerenders the loan form for the 
    // current user that has been logged in and then rerenders the loan form for the 
    // current user that has been logged in and then rerenders the loan form for the 
    // user to recreate a loan 
    // user to recreate a loan 
    // user to recreate a loan 
```

**Review by te4-alexander-nylundgomes — CHANGES_REQUESTED** [review]
Looks good, just remove the comments

**Review by te4-dennis-christensen — APPROVED** [review]
Looks good, just check Alexanders comment

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
Looks all good :+1:

**Pull request merged**